### ISSUE TITLE
java.lang.NoSuchMethodError: No static method in v2.0

### ISSUE THREAD

**Issue by lorenzovngl** [lorenzovngl/FoodExpirationDates#103]
### Description
[Version 2.0](https://github.com/lorenzovngl/FoodExpirationDates/tree/555676e836b4e9dfb4f18261d1c0d9d479a10d0b) is affected by a bug:
```
Fatal Exception: java.lang.NoSuchMethodError: No static method performImeAction$default(Landroidx/compose/ui/semantics/SemanticsPropertyReceiver;Ljava/lang/String;Lkotlin/jvm/functions/Function0;ILjava/lang/Object;)V in class Ln1/u; or its super classes (declaration of 'n1.u' appears in /data/app/~~O5S05GftgcRl67TcRjyO2A==/com.lorenzovainigli.foodexpirationdates-DAh1dEwCGimGJSX1GL1m8A==/base.apk)
```

### To Reproduce
Not exactly known yet. A user said that the bug occurs when he tried to insert a food item, so this suggest the problem comes from a textfield in the Insert Screen

### Smartphone
**The following data are extracted from Crashlitycs data, however only 27 of about 300 active users have faced this bug.**
 - Devices: Samsung Galaxy S23, Samsung Galaxy F23 5G, Fairphone 4, Samsung Galaxy S8+, MotoG84 5G, Pixel 7
 - OS: Android 14, 13, 12, 11, 10, 9
 - Version: 2.0

### Additional context
Full stacktrace:
```
Fatal Exception: java.lang.NoSuchMethodError: No static method performImeAction$default(Landroidx/compose/ui/semantics/SemanticsPropertyReceiver;Ljava/lang/String;Lkotlin/jvm/functions/Function0;ILjava/lang/Object;)V in class Ln1/u; or its super classes (declaration of 'n1.u' appears in /data/app/~~O5S05GftgcRl67TcRjyO2A==/com.lorenzovainigli.foodexpirationdates-DAh1dEwCGimGJSX1GL1m8A==/base.apk)
       at androidx.compose.foundation.text.CoreTextFieldKt$CoreTextField$semanticsModifier$1.invoke(CoreTextField.kt:184)
       at androidx.compose.foundation.text.CoreTextFieldKt$CoreTextField$semanticsModifier$1.invoke(CoreTextField.kt:1)
       at androidx.compose.ui.semantics.CoreSemanticsModifierNode.applySemantics(SemanticsModifier.kt:1)
       at androidx.compose.ui.node.LayoutNode$collapsedSemantics$1.invoke(LayoutNode.java:313)
       at androidx.compose.ui.node.LayoutNode$collapsedSemantics$1.invoke(LayoutNode.java:81)
       at androidx.compose.runtime.snapshots.Snapshot$Companion.observe(Snapshot.java:60)
       at androidx.compose.runtime.snapshots.SnapshotStateObserver$ObservedScopeMap.b(SnapshotStateObserver.kt:48)
       at androidx.compose.runtime.snapshots.SnapshotStateObserver.observeReads(SnapshotStateObserver.kt:153)
       at androidx.compose.ui.node.OwnerSnapshotObserver.observeReads$ui_release(OwnerSnapshotObserver.kt:1)
       at androidx.compose.ui.node.OwnerSnapshotObserver.observeSemanticsReads$ui_release(OwnerSnapshotObserver.kt:43)
       at androidx.compose.ui.node.LayoutNode.getCollapsedSemantics$ui_release(LayoutNode.kt:43)
       at androidx.compose.ui.semantics.SemanticsNodeKt.SemanticsNode(SemanticsNode.kt:110)
       at androidx.compose.ui.semantics.SemanticsNode.fillOneLayerOfSemanticsWrappers(SemanticsNode.kt:34)
       at androidx.compose.ui.semantics.SemanticsNode.fillOneLayerOfSemanticsWrappers(SemanticsNode.kt:42)
       at androidx.compose.ui.semantics.SemanticsNode.fillOneLayerOfSemanticsWrappers(SemanticsNode.kt:42)
       at androidx.compose.ui.semantics.SemanticsNode.fillOneLayerOfSemanticsWrappers(SemanticsNode.kt:42)
       at androidx.compose.ui.semantics.SemanticsNode.fillOneLayerOfSemanticsWrappers(SemanticsNode.kt:42)
       at androidx.compose.ui.semantics.SemanticsNode.fillOneLayerOfSemanticsWrappers(SemanticsNode.kt:42)
       at androidx.compose.ui.semantics.SemanticsNode.l(SemanticsNode.kt:15)
       at androidx.compose.ui.semantics.SemanticsNode.getChildren(SemanticsNode.kt:27)
       at androidx.compose.ui.semantics.SemanticsNode.getReplacedChildren$ui_release(SemanticsNode.kt:285)
       at androidx.compose.ui.platform.AndroidComposeViewAccessibilityDelegateCompat_androidKt.getAllUncoveredSemanticsNodesToMap$findAllSemanticNodesRecursive(AndroidComposeViewAccessibilityDelegateCompat.android.kt:285)
       at androidx.compose.ui.platform.AndroidComposeViewAccessibilityDelegateCompat_androidKt.getAllUncoveredSemanticsNodesToMap$findAllSemanticNodesRecursive(AndroidComposeViewAccessibilityDelegateCompat.android.kt:302)
       at androidx.compose.ui.platform.AndroidComposeViewAccessibilityDelegateCompat_androidKt.getAllUncoveredSemanticsNodesToMap$findAllSemanticNodesRecursive(AndroidComposeViewAccessibilityDelegateCompat.android.kt:302)
       at androidx.compose.ui.platform.AndroidComposeViewAccessibilityDelegateCompat_androidKt.getAllUncoveredSemanticsNodesToMap(AndroidComposeViewAccessibilityDelegateCompat_android.kt:76)
       at androidx.compose.ui.platform.AndroidComposeViewAccessibilityDelegateCompat_androidKt.access$getAllUncoveredSemanticsNodesToMap(AndroidComposeViewAccessibilityDelegateCompat_android.kt:76)
       at androidx.compose.ui.platform.AndroidComposeViewAccessibilityDelegateCompat.getCurrentSemanticsNodes(AndroidComposeViewAccessibilityDelegateCompat.android.kt:76)
       at androidx.compose.ui.platform.AndroidComposeViewAccessibilityDelegateCompat.sendAccessibilitySemanticsStructureChangeEvents(AndroidComposeViewAccessibilityDelegateCompat.android.kt:27)
       at androidx.compose.ui.platform.AndroidComposeViewAccessibilityDelegateCompat.checkForSemanticsChanges(AndroidComposeViewAccessibilityDelegateCompat.java:764)
       at androidx.compose.ui.platform.AndroidComposeViewAccessibilityDelegateCompat.semanticsChangeChecker$lambda$46(AndroidComposeViewAccessibilityDelegateCompat.java:764)
       at android.os.Handler.handleCallback(Handler.java:958)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loopOnce(Looper.java:230)
       at android.os.Looper.loop(Looper.java:319)
       at android.app.ActivityThread.main(ActivityThread.java:8893)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:608)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1103)
```

**Comment by lorenzovngl** [comment]
I think I found a discussion about this problem: [TextField Crashes if any accessibility service is enabled [322269946] - Issue Tracker](https://issuetracker.google.com/issues/322269946)

**Comment by lorenzovngl** [comment]
Solved with [version 2.0.1](https://github.com/lorenzovngl/FoodExpirationDates/releases/tag/v2.0.1)
